src/components/BasicHead.vue

```diff
@@ -104,21 +119,41 @@
     light
     >mdi-chevron-left</v-icon>
 
-  <v-divider vertical></v-divider>
-  <v-icon large class="iconStyle">mdi-printer</v-icon>
-  <v-divider vertical></v-divider>
-  <v-icon large class="iconStyle">mdi-bookmark-minus-outline</v-icon>
-  <v-divider vertical></v-divider>
-  <v-icon large class="iconStyle">mdi-folder-open-outline</v-icon>
-  <v-divider vertical></v-divider>
+  <v-divider vertical v-if="show('print')"></v-divider>
+
+  <v-btn v-if="tool.print.active && show('print')" 
+    large elevation="0" class="pa-1 ma-1" color="white" style="height: 55px">
+    <v-icon large class="iconStyle">mdi-printer</v-icon>
+  </v-btn>
+
+  <v-divider vertical v-if="show('print')"></v-divider>
+
+  <v-btn v-if="tool.bookmark.active && show('bookmark') && bookmarkVisible" 
+    large elevation="0" class="pa-1 ma-1" color="white" style="height: 55px" @click="showTags()">
+    <v-icon large class="iconStyle">mdi-bookmark-minus-outline</v-icon>
+  </v-btn>
+  <v-btn v-if="tool.bookmark.active && show('bookmark') && !bookmarkVisible" 
+    large elevation="0" class="pa-1 ma-1" color="white" style="height: 55px" disabled>
+    <v-icon large class="iconStyle">mdi-bookmark-minus-outline</v-icon>
+  </v-btn>
+
+  <v-divider vertical v-if="show('bookmark')"></v-divider>
+
+  <v-btn v-if="tool.collect.active && show('collect')" 
+    large elevation="0" class="pa-1 ma-1" color="white" style="height: 55px">
+    <v-icon large class="iconStyle">mdi-folder-open-outline</v-icon>
+  </v-btn>
+
+  <v-divider vertical v-if="show('collect')"></v-divider>
 
 </v-app-bar>
 </template>
 
 <style lang="scss">
-.iconStyle{
+.iconStyle {
```

Comment:
in general with this repo, we want to use onle `vxg-` prefixed styles, and export them globally so that they can be customized downstream

Reply:
Done